code/modules/mob/living/carbon/human/life.dm

```diff
@@ -905,8 +905,7 @@
 			spawn()
 				animate(src, pixel_y = pixel_y + 5 , time = 10, loop = 1, easing = SINE_EASING)
 			spawn(10)
-				if(flying)
-					animate(src, pixel_y = pixel_y - 5, time = 10, loop = 1, easing = SINE_EASING)
+				animate(src, pixel_y = pixel_y - 5, time = 10, loop = 1, easing = SINE_EASING)

```

Comment:
Oof, someone didn't understand how animations work.

You can take out **both** of the spawns above this (`animate` is an instant call, and spawning is expensive), and the `src, ` in this line, and it'll combine them into a single 2-second animation. As an added bonus, this will make sure `pixel_y` doesn't go wonky if something changes it on a flying person.

The end result will look like this:
```
if(flying)
	animate(src, pixel_y = pixel_y + 5 , time = 10, loop = 1, easing = SINE_EASING)
	animate(pixel_y = pixel_y - 5, time = 10, loop = 1, easing = SINE_EASING)
```